README.md

```diff
@@ -2,33 +2,46 @@
 
 # Kuzzle compatibility
```

Comment:
I would remove that part since the compatibility matrix will make it obsolete soon.